protostar/testing/environments/fuzz_test_execution_environment.py

```diff
@@ -59,6 +61,17 @@ async def execute(self, function_name: str) -> FuzzTestExecutionResult:
         database = InMemoryExampleDatabase()
         runs_counter = RunsCounter(budget=self.state.config.fuzz_max_examples)
 
+        if (
+            not self.state.config.fuzz_examples
+            and not self.state.config.fuzz_declared_strategies
+        ):
+            self._logger.warning(
+                "not providing the test data is deprecated and will break test cases in the future releases, "
+                "please use one of the following cheatcodes in the setup function in order to "
+                "explicitly provide data to be tested: \n- example\n- given"
+            )
+            # todo raise an error in future releases
```

Comment:
Can we create task for that on github instead of this TODO?

Reply:
comments tend to be forgotten :p

Reply:
#1008

Reply:
[remove todo comment](https://github.com/software-mansion/protostar/pull/995/commits/a087126c773de3184e464240861927027df7c323)